test-fork/CrossChainDeposits.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { increaseTime } from "./../test-utils/time";
+import { MockERC20 } from "./../types/generated/MockERC20";
+import { network } from "hardhat";
+import { expect } from "chai";
+import {
+    BaseRewardPool,
+    SiphonGauge__factory,
+    SiphonGauge,
+    SiphonDepositor,
+    SiphonDepositor__factory,
+    MockERC20__factory,
+    SiphonToken,
+    SiphonToken__factory,
+    BaseRewardPool__factory,
+    RAura__factory,
+    RAura,
+} from "../types/generated";
+import { BigNumberish, ethers, Signer } from "ethers";
+import { waitForTx } from "../tasks/utils";
+import { Phase2Deployed, SystemDeployed } from "../scripts/deploySystem";
+import { config } from "../tasks/deploy/mainnet-config";
+import { impersonate, impersonateAccount, simpleToExactAmount, ONE_WEEK } from "../test-utils";
+import { formatUnits } from "ethers/lib/utils";
+
+const debug = true;
+
+describe("Full Deployment", () => {
```

Comment:
This could probably just be a normal test file?

Reply:
Yeah potentially. Plan to get all this working and running then refactor the test files to neaten them up and break them up where it makes sense.